fuels-core/src/code_gen/functions_gen.rs

```diff
@@ -99,6 +96,7 @@ fn expand_fn_outputs(outputs: &[Property]) -> Result<TokenStream, Error> {
             }
         }
         _ => {
+            // There, everything will be expanded, which goes against the `1` case
```

Comment:
Do you mind removing this comment? IMO it would just make it more confusing for someone reading it without full context. We've already captured the tech debt as an issue, I think that's enough. Thoughts?